content/using/install.md

```diff
@@ -148,217 +108,133 @@ use |ota %disable or |ota ~sponsor %kids to reset it
 
-Because of their need for near-perfect uptime Stars should be hosted on cloud services. We have a guide for [hosting on DigitalOcean](@/using/operations/hosting.md), but any cloud provider should work.
+1. Start your planet. In the boot messages, look for a line that says something like `http: live (insecure, public) on 80`. The number given is the port that your planet is using.
+2.  If the port given is 80, simply type `localhost` into your browser's address bar. If the given port is a different number, such as `8080`, you would type `localhost:8080`. You'll be met with a login prompt.
+3. Type your password you got from running `+code` above.
+4. You're in! Now you can explore apps such as Chat, Publish, and more.
 
-Being an infrastructure node, a star has certain responsibilities to the Urbit network.
+### Join a group
 
-By default, your star accepts software updates from its galaxy and routes them to its planets. You can use this mechanism to push custom software to your planets. Keep in mind that planets expect functional, non-breaking software updates, and generally want to be able to communicate with planets that are sponsored by other stars.
+Now that you're on Landscape, join the Urbit Community, a great place for newcomers to ask questions.
 
-To ensure new planets can connect to your ship, users are expected to participate in network-wide breaches by updating to the latest Urbit version, deleting (or archiving) your pier, and then booting your ship using the new binary. If you don’t participate, you won’t be able to communicate with anyone on the network who has updated to the new era.
+Click the 'Groups' button on your Landscape home page, and then click the `Join Group` column. Then, enter `~bitbet-bolbel/urbit-community` into the field and press the `Join Group` button.
 
-Network-wide breaches are distinct from personal breaches, wherein an individual ship cycles its personal network keys using bridge.
+From the Urbit Community group you can join a variety of chatrooms, notebooks, and collections.
 
-See our [Guide to Breaches](@/docs/tutorials/guide-to-breaches.md) for more information and for instructions on breaching.
+## Next Steps
 
-### Running a galaxy
+Now that you’re up and running, take some time to deepen your understanding by checking out some of our other guides:
 
-A galaxy is at the top of the hierarchy in terms of importance. Thus, if you're interested in running one, please contact support@urbit.org, and we will give you personalized assistance in getting set up.
+- [Host your Ship in the Cloud](@/using/operations/hosting): Follow this guide to set your ship up to run as a cloud server, accessible from anywhere.
+- [Read the Operations Manual](@/using/operations/using-your-ship): Become an expert pilot by learning to use your ship.
+- [Learn to Develop on Urbit](@/using/develop): Learn how to contribute to or develop on Urbit.
```

Comment:
It's failing 'cause of these. They need to end in .md — they point to the files in our zola directory here, not to the permalink location.